test/test.js

```diff
@@ -9,25 +12,24 @@ const is = require("@slimio/is");
 const prettyJSON = require("../");
 
 async function executeNodeScript(path) {
-    const { stdout } = spawn(process.argv[0], [path]);
-    let str = "";
-    for await (const buf of stdout) {
-        str += buf;
-    }
+  const { stdout } = spawn(process.argv[0], [path]);
+  let str = "";
+  for await (const buf of stdout) {
+    str += buf;
+  }
 
-    str | 0;
-    return str;
-}
+  // eslint-disable-next-line no-unused-expressions
+  str | 0;
 
-test("prettyJSON must be a function", () => {
-    expect(is.func(prettyJSON)).toStrictEqual(true);
-});
+  return str;
+}
 
-test("prettyJSON of non-object must return undefined", () => {
-    expect(prettyJSON("hello")).toStrictEqual(undefined);
-});
+describe("getLocalLang/setLocalLang", () => {
+  it("prettyJSON must be a function", () => {
+    assert.strictEqual(is.func(prettyJSON), true);
+  });
 
-test("prettyJSON stdout must be the same", async() => {
-    const stdout = await executeNodeScript(join(__dirname, "fixtures", "01.js"));
-    expect(stdout).toMatchSnapshot();
+  it("prettyJSON of non-object must return undefined", () => {
+    assert.strictEqual(prettyJSON("hello"), undefined);
```

Comment:
This test is wrong as `prettyJSON` will never return anything.
```js
assert.strictEqual(prettyJSON({ valid: 'json' }), undefined);
```
This test would also pass.
Maybe the function should throw in case we have anything else than `Object` or `Array` and test it ?
Something like
```js
if (is.object(obj)) {
    (Array.isArray(obj) ? logArray : logObject)(obj);
    process.stdout.write(EOL);
} else {
  throw TypeError('.... should be object or array')
}
```
(it's just an idea, maybe its simpler to keep this test or remove it)

Reply:
In this test, the return value of `prettyJSON` is `undefined`

Reply:
Yes, but `prettyJSON` will never return a value, it mean it doesn't return `undefined` **because** of the non object value. It will anyway return `undefined`, that why this test is wrong for me